nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/pom.xml

```diff
@@ -107,6 +146,38 @@ language governing permissions and limitations under the License. -->
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
I think we talked about this before but can't find it. Does it need to be here? I thought we can inherit the one from the top-level parent POM?